### DISCUSSION TITLE
Network drive as local drive?

### DISCUSSION THREAD

**Discussion by geffao** [dokan-dev/dokany#962] [Ideas]
Hi there folks! Just cross upon this project when trying to figure a way to make a network drive to behave as an local internal drive. So yes, I have been trying to do just this, but couldn't find a way to properly do it. I have tried the command "mirror.exe /m /r Z:\ /l y but haven't been successful. The drive shows up, but on Command Prompt it never says it was successful and as you see, it doesn't show no file system in it, and I get an error if I try to open it.

Lyrina, the moderator here told this:
mirror.exe /r Z:\ /l y
This should be enough but the issue you face is that the mirror.cc code does not support to mirror network path.
I believe small changes would make it work without much issues.

So I leave as an idea, if possible to be done. That would be amazing to get some softwares that don't recognize network shares to work.

![image](https://user-images.githubusercontent.com/77840067/105526428-138cda00-5ceb-11eb-8f59-65781fc2d1d3.png)

![image](https://user-images.githubusercontent.com/77840067/105526442-18ea2480-5ceb-11eb-81c1-8cbbf0bfbd37.png)

**Comment by Xhado7** [comment]
I would love to have this feature as well.

**Comment by PacFox** [comment]
I've been normally mounting the network share directly. Does `mirror.exe /r \\msedgewin10\share /l y` work for your case?

**Reply by Gillous** [reply]
It worked well for me! Amazing!!!

**Reply by BolverBlitz** [reply]
For me that only works for about 30s. 
All files are are working in those 30s
![grafik](https://user-images.githubusercontent.com/35345288/143618270-4091ed98-df6f-43ff-8f44-3b6e82f6f537.png)

**Comment by Daniel-Abrecht** [comment]
I usually create a symlink on my NTFS drive to my network shares using "mklink". There are some quirks to this in windows which make things behave almost as if it was a local drive, if accessed over that path. It can also trick most programs into not noticing that it isn't local.
